sdk/monitor/monitor-opentelemetry/src/logs/handler.ts

```diff
@@ -55,14 +56,29 @@ export class LogHandler {
    * Start auto collection of telemetry
    */
   private _initializeInstrumentations() {
-    if (this._config.instrumentationOptions.bunyan?.enabled) {
+    const bunyanLogLevelEnv =
+      process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL || process.env.BUNYAN_LOG_LEVEL;
+    const winstonLogLevelEnv =
+      process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL ||
+      process.env.WINSTON_LOG_LEVEL;
+
+    // No Severity Number for no bunyan logging, so disable instrumentation if NONE is set.
+    if (this._config.instrumentationOptions.bunyan?.enabled && bunyanLogLevelEnv !== "NONE") {
```

Comment:
Where the "NONE" value being set?, is NONE a valid logging level in Java?

Reply:
Java has a corresponding "OFF" setting that "NONE" here corresponds to.

Reply:
We do not need a "None" or "off" setting, this can be configured using "instrumentationOptions" through JSON config

Reply:
Makes sense. Configuring log level to "NONE" would overlap with the same functionality as simply disabling the instrumentation. Cleaning up this check.